app/components/Analyst/Project/ProjectInformation/ProjectInformationForm.tsx

```diff
@@ -44,6 +86,87 @@ const ProjectInformationForm = ({ application }) => {
 
   const handleSubmit = (e) => {
     e.preventDefault();
+    if (
+      !formData.hasFundingAgreementBeenSigned &&
+      projectInformation.applicationByApplicationId
+    ) {
+      // archive sows
+      projectInformation.applicationByApplicationId.applicationSowDataByApplicationId.nodes.forEach(
+        (node: {
+          id: any;
+          archivedAt: string | null;
+          sowTab1SBySowId: { nodes: any[] };
+          sowTab2SBySowId: { nodes: any[] };
+          sowTab7SBySowId: { nodes: any[] };
+          sowTab8SBySowId: { nodes: any[] };
+        }) => {
+          const archivedAt = new Date().toISOString();
+          const applicationSowInput = {
+            id: node.id,
+            applicationSowDataPatch: { archivedAt },
+          };
+          if (node.archivedAt === null) {
+            archiveSowData({
+              variables: {
+                input: applicationSowInput,
```

Comment:
We usually create a mutation to archive on creation of another record, to keep it all as part of one transaction to prevent any network errors preventing the DB from being in an undefined state (tabs 1 and 2 are archived, but the request for 7 and 8 didn't go through for example). 

Not exactly the case here, but would be a good idea to create a "saveProjectInformationAndArchive" db mutation function to handle updates in the future as well. Right now we're using the `set_archived` trigger function on some places to automatically archive when we create a new record - can use that for the project_information and the sow_data table but the individual tabs will need some logic in the function itself